lib/xcmv/file.rb

```diff
@@ -50,10 +50,28 @@ def create_file_reference
       end
     end
 
+    def infer_target_membership
+      # attempt to infer target membership from sibling files
```

Comment:
Can you move this logic into a method like `GroupMembership#inferred_targets`?  I think this logic makes more sense there (since `GroupMembership` is already doing nearby-target lookup).

Then it probably makes sense to move the fallback logic into `#add_to_targets`.